### PR TITLE
Stop container that is started out of band

### LINKED ISSUE #1398: Unable to stop container powered on by UI
```
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls start 646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064
646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps
CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS               NAMES
646e745164a7        busybox             "top"               Less than a second ago   Running                                 runk_easley
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls stop 646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064
646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS               NAMES
646e745164a7        busybox             "top"               Less than a second ago   Stopped                                 runk_easley
vagrant@devbox:~/vic$ # POWER ON THE CONTAINER VIA VSPHERE UI
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps
CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS               NAMES
646e745164a7        busybox             "top"               Less than a second ago   Running                                 runk_easley
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls stop 646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064
Error response from daemon: container 646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064 not found
Error: failed to stop containers: [646e745164a74834ab269a86fb6a6d3e28c95f99383940121863a5380445a064]
vagrant@devbox:~/vic$
```

### PR DESCRIPTION
**Pull request by cgtexmex**
This fixes the issue of being unable to stop a container that
is started out of band -- possibly via the vSphere UI or govc.

Fixes #1398

### CODE REVIEW

**Comment by jzt** [comment]
lgtm

File: lib/apiservers/engine/backends/container.go [hunk 1]
```diff
@@ -790,37 +790,41 @@ func (c *Container) containerStop(name string, seconds int, unbound bool) error
 	}
 
 	if unbound {
+		// assume that we have a network configured
+		networkFound := true
 		// get the endpoints for the container
 		conEpsRes, err := client.Scopes.GetContainerEndpoints(scopes.NewGetContainerEndpointsParams().WithHandleOrID(handle))
 		if err != nil {
 			switch err := err.(type) {
 			case *scopes.GetContainerEndpointsNotFound:
```
Comment (hmahmood) [inline lib/apiservers/engine/backends/container.go]: I would prefer if you took this error out and modified GetContainerEndpoints in the portlayer to return an empty array of endpoints in this case. The code will be cleaner.

Comment (cgtexmex) [inline lib/apiservers/engine/backends/container.go]: OK - and then if there are no endpoints don't call the other functions -- correct?

**Comment by cgtexmex** [comment]
After a discussion with @hmahmood I'm closing this and assigning the issue to him.

**Pull request closed without merging**